test/sync/server-not-available.js

```diff
@@ -0,0 +1,90 @@
+require('chai').should();
+
+const {
+  initClient,
+  performQuery,
+  performWatchQuery,
+  performOfflineMutation
+} = require('../../util/sync-client');
+const SyncServer = require('../../util/sync-server');
+const SyncBrowser = require('../../util/sync-browser');
+
+describe('Data Sync - Server not available', function() {
+  this.timeout(0);
+
+  let syncServer;
+  let syncBrowser;
+
+  let watchPromise;
+
+  before('start browser client', async function() {
+    syncBrowser = new SyncBrowser();
+    await syncBrowser.start();
+  });
+
+  after('close browser client', async function() {
+    await syncBrowser.stop();
+  });
+
+  it('should run the voyager server', async function() {
```

Comment:
Using `it` for each step is it a good practice? We are not doing this in any other tests, maybe we should decide all together if using this kind of syntax and then use a standard way in all our tests